pandas/_libs/index.pyx

```diff
@@ -301,7 +301,7 @@ cdef class IndexEngine:
                 values = self.values
                 self.monotonic_inc, self.monotonic_dec, is_strict_monotonic = \
                     self._call_monotonic(values)
-            except (TypeError, InvalidOperation):
+            except (TypeError, InvalidOperation, ValueError):
```

Comment:
can we test this somehow? We have "test_downstream.py" for this?

Reply:
would you be ok with just testing against a pyarrow list type? e.g. this raises a `ValueError`:

```
import pyarrow as pa
import pandas as pd

arr = pd.array([[1], [2, 3]], dtype=pd.ArrowDtype(pa.list_(pa.int64())))

pd.Index(arr).is_monotonic_increasing  # ValueError
```

Reply:
Yep that’s fine

Reply:
I just want to avoid that someone removed the ValueError and the tests don’t fail

Reply:
thanks, test added